odoo/addons/base/models/ir_model.py

```diff
@@ -1227,6 +1234,43 @@ def write(self, vals):
 
         return result
 
+    @api.model
+    def __update_reflected_field(self, table, field, value, selection_value):
+        query = 'UPDATE "{table}" SET "{field}"=%s WHERE "{field}"=%s'.format(
+            table=table, field=field,
+        )
+        self.env.cr.execute(query, (value, selection_value,))
+
+    @api.model
+    def __delete_reflected_records(self, table, field, selection_value):
+        query = 'DELETE FROM "{table}" WHERE "{field}"=%s'.format(
+            table=table, field=field,
+        )
+        self.env.cr.execute(query, (selection_value,))
+
+    def _handle_ondelete(self):
+        for selection in self:
```

Comment:
cf odoo/odoo#47665 ensure abstract models are handled (and why did you removed the `selection.field_id.store` part?)

Reply:
probably a mistake on my part, I'll add a test for it